.github/CONTRIBUTING.md

```diff
@@ -2,72 +2,73 @@
 
 Contributions are welcome! Here's how you can help:
 
-- [Contributing code](#code)
-- [Reporting issues](#issues)
-- [Requesting features](#feature-requests)
-- [Translating](#translations)
-- [Donating](#donations)
+* [Contributing code](#code)
+* [Reporting issues](#issues)
+* [Requesting features](#feature-requests)
+* [Translating](#translations)
+* [Donating](#donations)
 
 ## Code
 
 1. [Fork](https://help.github.com/articles/fork-a-repo/) the repository and
    [clone](https://help.github.com/articles/cloning-a-repository/) your fork.
 
 2. Before you start coding, consider opening an
-   [issue at Github](https://github.com/minetest/minetest/issues) to discuss the
+   [issue at GitHub](https://github.com/minetest/minetest/issues) to discuss the
    suitability and implementation of your intended contribution with the core
    developers.
 
-   Any Pull Request that isn't a bug fix and isn't covered by
+   Any pull request that isn't a bug fix and isn't covered by
    [the roadmap](../doc/direction.md) will be closed within a month unless it
    receives a concept approval from a Core Developer. For this reason, it is
    recommended that you open an issue for any such pull requests before doing
    the work, to avoid disappointment.
 
    You may also benefit from discussing on our IRC development channel
-   [#minetest-dev](http://www.minetest.net/irc/). Note that a proper IRC client
```

Comment:
This page doesn't exist anymore and redirects to `/get-involved/`, https://wiki.minetest.net/IRC would probably be a better page to link to.

Reply:
Done, and reworded a bit.